tests/LineBot.Tests/RichMenuRequestTests/TheRichMenuSizeClassTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
```

Comment:
This file should be move to `tests/LineBot.Tests/RichMenu/RichMenuSizeTests/` and split into a separate file for each property.